### PR TITLE
docs: add base en

### PR DESCRIPTION
**Pull request by xrkffgg**
- 英文版
   - [x] 基础使用

---
### 中文改动

调整了一下 中文 modal 的 Demo 
新增了 Alert  默认 type 

---
### 问题

1. 其中有一张图片，没有英文版本。
2. 中英文主题开发链接未配置。

### ref

https://github.com/umijs/dumi/pull/419

### CODE REVIEW

File: docs/.demos/modal/modal.jsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /**
- * title: 基础 Modal
- * desc: 这是 antd Modal 组件的基础示例
+ * title: Basic Modal
+ * desc: This is a basic example of the antd Modal component
```
Comment (PeachScript) [inline docs/.demos/modal/modal.jsx:3]: 这里可以用 `title.zh-CN` 以及 `desc.zh-CN` 来复用组件

File: docs/guide/basic.md [hunk 2]
```diff
@@ -1,17 +1,495 @@
...
+
+By default, dumi will search for the Markdown documents of all sub-packages and generate routes based on the path `packages/[package name]/src`.
+
+There is currently no scaffolding for the lerna project. You can check [ahooks](https://github.com/alibaba/hooks) project as a reference.
+
+## Conventional routing
+
+As shown in the figure below, dumi's conventional routing rules are very simple:
+
+<img src="https://gw.alipayobjects.com/zos/bmw-prod/e607ec1b-8f5d-4fec-9557-42298ceab02e/kiimo7of_w1410_h548.png" width="705" />
```
Comment (PeachScript) [inline docs/guide/basic.md:198]: 图片文案求翻译，我重新搞一张英文的

File: docs/guide/basic.md [hunk 3]
```diff
@@ -1,17 +1,495 @@
...
+<embed src="/path/to/some.md"></embed>
+
+<!-- Introduce the Markdown file content of the specified line according to the line number -->
+<embed src="/path/to/some.md#L1"></embed>
+
+<!-- Introduce part of the Markdown file content based on the line number -->
+<embed src="/path/to/some.md#L1-L10"></embed>
+```
+
+We can also use the theme API provided by dumi to replicate and add built-in components. Visit [Theme-Theme Development]() to learn more.
```
Comment (PeachScript) [inline docs/guide/basic.md:495]: 链接我加漏了，文档是已经有了
```suggestion
We can also use the theme API provided by dumi to replicate and add built-in components. Visit [Theme-Theme Development](/theme/development) to learn more.
```

**Comment by xrkffgg** [comment]
磁盘目录  Disk directory
基础路径  Base path

The path in the resolve.includes configuration item is the base path. Dumi will search for markdown files from these paths, and these paths will not participate in route generation.

前缀路由  Prefix routing

The path before the Markdown file. In doc mode, it will be recognized as a group route as a whole. In site mode, the first-level path will be recognized as navigation routing, and the overall path will be recognized as group routing.

文件路由   File routing

The name of the Markdown file. It will form the default page route together with the prefix routing. If the file name is index.md, the page routing is equivalent to the prefix routing.

File: docs/guide/basic.md (hunk 2 above)
Comment (PeachScript) [inline docs/guide/basic.md:198]:
```suggestion
<img src="https://gw.alipayobjects.com/zos/bmw-prod/66ca1c57-3146-40d5-bea4-24dea86eb95b/kijoyzad_w1540_h732.png" width="770" />
```

File: docs/guide/basic.zh-CN.md [hunk 4]
```diff
@@ -473,4 +473,4 @@ dumi 对 HTML 默认的 `embed` 标签做了扩展，可以在一个 Markdown 
 <embed src="/path/to/some.md#L1-L10"></embed>
 ```
 
-我们也可以通过 dumi 提供的主题 API 实现内置组件的复写和新增，可访问 [主题 - 主题开发]() 了解更多。
+我们也可以通过 dumi 提供的主题 API 实现内置组件的复写和新增，可访问 [主题 - 主题开发](/theme/development) 了解更多。
```
Comment (PeachScript) [inline docs/guide/basic.zh-CN.md:476]:
```suggestion
我们也可以通过 dumi 提供的主题 API 实现内置组件的复写和新增，可访问 [主题 - 主题开发](/zh-CN/theme/development) 了解更多。
```

File: docs/guide/basic.md [hunk 5]
```diff
@@ -1,17 +1,495 @@
...
+    I only show in the development environment
+  </Previewer>
+);
+```
+
+#### iframe mode
+
+Setting `iframe` to `true` will use `iframe` to render the demo, which can be completely isolated from the document.
+
+It is usually used for layout components. In this case, [`compact`](/zh-CN/config/frontmatter# compact) The configuration defaults to `true`:
```
Comment (PeachScript) [inline docs/guide/basic.md:433]:
```suggestion
It is usually used for layout components. In this case, [`compact`](/config/frontmatter#compact) The configuration defaults to `true`:
```

**Review by PeachScript — APPROVED** [review]

**Pull request merged**